UI/styles/admin.css

```diff
@@ -0,0 +1,193 @@
+#gov-office{
+    margin-top: 25px;
+}
+
+#head{
+    font-size: 1.8rem;
+    font-weight: 400;
+    margin: 0;
+    margin-top: 1rem;
+    margin-bottom: 3rem;
+    text-align: center;
+    color: black;
+}
+
+#head2{
+    color: black;
+    font-weight: bold;
+    letter-spacing: 1px;
+    margin: .5rem 0;
+}
+
+#party-form {
+    max-height: 300px;
+    margin: 2rem auto;
+    width: 600px;
+}
+
+input {
+    display: block;
+    width: 100%;
+    border: 3px solid silver;
+    padding: 5px;
+    font-size: 18px;
+    border-radius: 10px;
+    margin: 3px;
+}
+
+table {
+    font-family: Arial, Helvetica, sans-serif;
+    border-collapse: collapse;
+    width: 100%;
+}
+
+td,th {
+    border: 2px solid silver;
+    text-align: left;
+    padding: 5px;
+}
+
+tbody td {
+    text-align: center;
+}
+
+#submitButton {
+    margin-top: 10px;
+}
+
+#isbn{
+    margin: 0;
+    padding: 0;
+    width: 30px;
+}
+
+#logo-head{
+    margin: 0;
+    padding: 0;
+    width: 30px;
+}
+
+#logo1{
+    color: black;
+    font-size: 2em;
+    margin:0;
+}
+
+.success, .error {
+    color: white;
+    padding: 5px;
+    margin: 5px 0 15px o;
+}
+
+.success {
+    background-color: green;
+}
+
+.error {
+    background-color: red;
+}
+
+
+
+  
+  /* The dropdown container */
+  .dropdown {
+    overflow: hidden;
+  }
+  
+  /* Dropdown button */
+  .dropdown .dropbtn {
+    font-size: 25px; 
+    border: none;
+    outline: none;
+    color: green;
+    background-color: inherit;
+    font-family: inherit; /* Important for vertical align on mobile phones */
+    margin: 0; /* Important for vertical align on mobile phones */
+  }
+  
+  /* Add a red background color to navbar links on hover */
+  .navbar a:hover, .dropdown:hover .dropbtn {
+   /* background-color: green;*/
+    font-weight: 600;
```

Comment:
Remove commented or unused code

Reply:
Implemented feedback on this already, PR request will be sent soon
Thanks alot